### PR TITLE
Participant swap

### PR DESCRIPTION
**Pull request by swapnils118**
Fixes: #39 

## Description
Tasks done:
- Folder structure of Participant page
- Individual component skeleton created
- Questionnaire Completion component created.

## Feature Link:
- Questionnaire Completion component

## Description:
Added components in participant folder/page and bare skeleton created for each individual component. Questionnaire completion component created which is one of the components of participant page. Added all the styles and classes as per requirement.

---

## Screenshots:
participantPageComponents
questionnaireCompletionComponent

---

## 📢 Notes for Reviewers
Add any additional information or context for the reviewers.

### CODE REVIEW

File: src/components/questionnaire-completion.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        </div>
+      </section>
+      <footer className=" flex justify-between mt-6 mx-2 w-[370px]">
+        <small>
+          <u className="text-white">Download</u>
+        </small>
+        <small>
+          <u className="text-white">View all</u>
+        </small>
+      </footer>
```
Comment (oveed) [inline src/components/questionnaire-completion.tsx:33]:
```suggestion
     <footer className=" flex justify-between mt-6 mx-2 w-[370px] text-white underline">
      <small>
          <a >Download</a>
        </small>
        <small>
          <a >View all</a>
        </small>
      </footer>
```

File: src/pages/participant-profile/index.tsx [hunk 2]
```diff
@@ -1,14 +1,29 @@
 import { FunctionComponent } from 'react';
 import { UpcomingSession } from './components/upcoming-block';
+import Progress from '../../components/progress';
+import PerformanceMeasure from '../../components/performance-measure';
+import ProfileSummary from '../../components/profile-summary';
+import ProgramInformation from '../../components/program-information';
+import QuestionnaireCompletion from '../../components/questionnaire-completion';
```
Comment (oveed) [inline src/pages/participant-profile/index.tsx:7]: I think these should go in the participant profile components

File: src/pages/participants/style.css
Comment (oveed) [inline src/pages/participants/style.css:1]: Could you move this to the participant profile style file, participants is another page.

**Review by oveed — CHANGES_REQUESTED** [review]
Everything else looks good, well done.

File: src/pages/participants/style.css
Comment (swapnils118) [reply src/pages/participants/style.css:1]: Yes for sure.
You want me to create new route for it?

Comment (swapnils118) [reply src/pages/participants/style.css:1]: Participants

ATTENTION PLEASE:
Isn't this upcoming block component the same in this design model and the block present in /participants/:id route(which is the block above the one I created?)

**Review by oveed — APPROVED** [review]

File: src/pages/participants/style.css
Comment (oveed) [reply src/pages/participants/style.css:1]: Yes i created it, we're working on the same page.

**Review by oveed — APPROVED** [review]

**Comment by tianchengc** [comment]
@swapnils118 there is merge conflict, could you fix it

**Pull request merged**